MAUI/Themes/Keys.md

```diff
@@ -294,4 +294,1497 @@ This page lists the keys for each control and the element to which it is mapped
         <td> SfExpanderPressedHeaderIconColor <br/><br/></td> 
         <td> Color of header icon when it is in pressed state.<br/><br/></td>
     </tr>
+</table>
+
+## SfCalendar
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+    <tr>
+        <td rowspan="55">
+            SfCalendarStyles
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfCalendarViewTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, it is possible to customize the appearance of the SfCalendarTheme without merging common theme resource and control style resource dictionaries.
```

Comment:
By merging this key in application resources, it is possible to customize the appearance of the SfCalendarTheme without merging common theme resources and control style resource dictionaries.